### PR TITLE
Fix handling of token renewal

### PR DESCRIPTION
**Pull request by mattwire**
Ok, this fixes the access token issues in the latest version

### CODE REVIEW

File: CRM/Civixero/OAuth2/TokenStoreDefault.php [hunk 1]
```diff
@@ -13,22 +13,39 @@ class CRM_Civixero_OAuth2_TokenStoreDefault implements CRM_Civixero_OAuth2_Token
 
   protected $token;
 
+  protected $connectorID;
+
+  /**
+   * Get the CiviCRM AccountSync Connector ID
+   * @return mixed
+   */
+  public function getConnectorID() {
```
Comment (eileenmcnaughton) [inline CRM/Civixero/OAuth2/TokenStoreDefault.php:22]: Can we return hint this?

**Comment by eileenmcnaughton** [comment]
Thanks @mattwire  - hopefully one of the people who reported the issue can test it

**Comment by eileenmcnaughton** [comment]
This behaves the same for me as without it - ie false negatives on the system check & works on the sync. I actually have connectors configured so that might be why I seem to have less problems. Anyway - given this doesn't break it for me & others have it actively broken & the test still runs I'm gonna merge this so that the others can better test

**Comment by eileenmcnaughton** [comment]
Hmm - I take it back - now the form is throwing the dreaded `invalid_grant` exception for me...

**Comment by eileenmcnaughton** [comment]
still I think it might be better to have this merged & then fix on top of it as there is some re-org in this

**Comment by eileenmcnaughton** [comment]
Dang - now my authorization is totally broken on the site I put this on

**Comment by eileenmcnaughton** [comment]
Phew got it working - it was the url - I just need to see if it needs to keep resetting now ....

File: CRM/Civixero/OAuth2/TokenStoreDefault.php (hunk 1 above)
Comment (mattwire) [reply CRM/Civixero/OAuth2/TokenStoreDefault.php:22]: Here you go: https://github.com/eileenmcnaughton/nz.co.fuzion.civixero/pull/127

**Pull request merged**